src/css/my-widgets.scss

```diff
@@ -389,28 +389,55 @@
 		opacity: 0.5;
 	}
 
-	.page .controls div.delete_dialogue {
-		background: url(../../../img/delete-dialogue-bg.png) no-repeat center;
-		padding: 20px 0;
-		margin-bottom: 5px;
+	.page .controls .delete_dialogue {
 		text-align: center;
-	}
+		width: 345px;
+		margin: 0;
+		position: absolute;
+		padding: 21px 10px 17px;
+		box-sizing: border-box;
+		background: white;
+		border-radius: 10px;
+		box-shadow: 3px 1px 6px rgba(0, 0, 0, 0.3);
+
+		// css triangle arrow to point at delete link
+		&:before {
+			content: ' ';
+			height: 0;
+			position: absolute;
+			width: 0;
+			top: -35px;
+			right: 20px;
+			border: 20px solid transparent;
+			border-bottom-color: white;
+		}
 
-	.page .controls div.delete_dialogue .action_button {
-		margin-top: 5px;
-	}
+		.bottom_buttons {
+			margin-top: 18px;
+		}
 
-	.page .controls div.delete_dialogue .red {
-		background: #ca0000;
-		background-image: linear-gradient(#e10000, #ca0000);
-		border-color: #747474;
-		color: #ffffff;
-	}
+		.action_button {
+			margin-top: 5px;
+		}
 
-	.page .controls div.delete_dialogue .red:hover {
-		background: #ca0000;
-		background-image: linear-gradient(#ca0000, #e10000);
-		text-decoration: none;
+		.red {
```

Comment:
A while ago these files were ported over from raw css to scss with the intent of improving them gradually.  Most of them still have redundant & plain css, so please upgrade them as you move through.